### DISCUSSION TITLE
Unable to use gpt4free for text editing

### DISCUSSION THREAD

**Discussion by SergeOd1** [xtekky/gpt4free#1203] [General]
I made a bot that edits the article and publishes it in telegram, but GPT sometimes inserts comments and spoils everything. I've tried different "don't comment" options, but it doesn't help.
And I wanted to make the bot completely independent, I think many have faced this

**Comment by TheGreatApollyon** [comment]
You should elaborate on what exactly you mean by "independent". And gpt returning the articles with comments are not caused or controlled by g4f.
